packages/components/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Bug Fixes
 
 -   `Button`: Never apply `aria-disabled` to anchor ([#63376](https://github.com/WordPress/gutenberg/pull/63376)).
+-   `ToggleGroupControl`: support disabled options ([#63450](https://github.com/WordPress/gutenberg/pull/63450)).
```

Comment:
Is this really a bug fix? Sounds like an enhancement to me.

Reply:
My train of thoughts was that we never stated that `disabled` options were not supported — so, from a consumer's point of view, this fixes `disabled` support.

But enhancement works too — I will change that.